cswinrt/strings/additions/Windows.Storage.Streams/StreamOperationsImplementation.cs

```diff
@@ -201,8 +201,9 @@ internal static class StreamOperationsImplementation
                     int buffSize = 0x4000;
```

Comment:
To confirm, WriteAsync in the if part doesn't have issues with 0 bytesToWrite right?

Reply:
Do you mean CopyToAsync? It does, it is not designed to support 0 bytes

Reply:
I was referring to a couple lines above:                     await stream.WriteAsync(data, offset, bytesToWrite, cancelToken).ConfigureAwait(continueOnCapturedContext: false);

Reply:
Yes, I believe this method is testing that WriteAsync: https://github.com/microsoft/CsWinRT/blob/3a28fb4fcc240bae8f89145fb028e2b63226d596/UnitTest/TestComponentCSharp_Tests.cs#L337

I added the same line but specified writing 0 bytes to front and end of the array, and no errors were thrown. 

I'm adding a unit test for that as well